Packs/Slack/Integrations/Slack_IAM/Slack_IAM.py

```diff
@@ -138,6 +238,284 @@ def get_mapping_fields(client: Client) -> GetMappingFieldsResponse:
+        "nameFamilyName": "name.familyName",
+        "nameGivenName": "name.givenName",
+        "nameHonorificPrefix": "name.honorificPrefix",
+        "nameHonorificSuffix": "name.honorificSuffix",
+        "nameMiddleName": "name.middleName",
+        "nickName": "nickName",
+        "organization": scim_extension + ".organization",
+        "password": "password",
+        "photo": "photos(val.type && val.type=='photo').[0].value",
+        "preferredLanguage": "preferredLanguage",
+        "profileUrl": "profileUrl",
+        "thumbnnail": "photos(val.type && val.type=='thumbnail').[0].value",
+        "timezone": "timezone",
+        "title": "title",
+        "userName": "userName",
+        "userType": "userType",
+    }
+    ret = dict()
+    for k, v in mapping.items():
+        try:
+            ret[k] = demisto.dt(clientData, v)
+        except Exception:
+            ret[k] = None
+    return ret
+
+
+def verify_and_load_scim_data(scim):
+    scim = json.loads(scim)
+    if type(scim) != dict:
+        raise Exception('SCIM data is not a valid JSON')
+    return scim
+
+
+def get_group_command(client, args):
+    scim = verify_and_load_scim_data(args.get('scim'))
+
+    group_id = scim.get('id')
+    group_name = scim.get('displayName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'id' or 'displayName' in the scim data")
+    if not group_id:
+        res = client.search_group(group_name)
+        res_json = res.json()
+
+        if res.status_code == 200:
+            if res_json.get('totalResults') < 1:
+                generic_iam_context = OutputContext(success=False, displayName=group_name, errorCode=404,
+                                                    errorMessage="Group Not Found", details=res_json)
+                return CommandResults(
+                    raw_response=generic_iam_context.data,
+                    outputs_prefix=generic_iam_context.command,
+                    outputs_key_field='id',
+                    outputs=generic_iam_context.data,
+                    readable_output=tableToMarkdown('Slack Get Group:', generic_iam_context.data, removeNull=True)
+                )
+            else:
+                group_id = res_json['Resources'][0].get('id')
+        else:
+            generic_iam_context = OutputContext(success=False, displayName=group_name, id=group_id,
+                                                errorCode=res_json['Errors']['code'],
+                                                errorMessage=res_json['Errors']['description'], details=res_json)
+            return CommandResults(
+                raw_response=generic_iam_context.data,
+                outputs_prefix=generic_iam_context.command,
+                outputs_key_field='id',
+                outputs=generic_iam_context.data,
+                readable_output=tableToMarkdown('Slack Get Group:', generic_iam_context.data, removeNull=True)
+            )
```

Comment:
It won't be that readable here, I'll add it as a function for the other cases, in case no id is provided we will search by name, but in this function we want to catch different errors - such as if no group was found or other errors.